neo4j/db_manipulation/add_notification_from_report_to_filing_user.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+ENV_FILE=$(dirname "$0")/.env
+[[ -f "$ENV_FILE" ]] && source "$ENV_FILE"
+
+if [ -z "$NEO4J_USERNAME" ] || [ -z "$NEO4J_PASSWORD" ]; then
+  echo "Please set NEO4J_USERNAME and NEO4J_PASSWORD environment variables."
+  echo "Database manipulation is not possible without connecting to the database."
+  echo "E.g. you could \`cp .env.template .env\` unless you run the script in a docker container"
+fi
+
+until echo 'RETURN "Connection successful" as info;' | cypher-shell
+do
+  echo "Connecting to neo4j failed, trying again..."
+  sleep 1
+done
+
+echo "
+  :begin
+  MATCH (submitter:User)-[filed:FILED]->(report:Report)
+  WHERE NOT (submitter)<-[:NOTIFIED]-(report)
+  CREATE (submitter)<-[notification:NOTIFIED]-(report)
+  SET notification.createdAt = filed.createdAt,
+    notification.updatedAt = notification.createdAt,
+    notification.read = FALSE,
+    notification.reason = 'filed_report_on_resource';
+  :commit
+" | cypher-shell
```

Comment:
I think we can remove the first solution :point_down: since it wasn't a working solution, and we never used it.
maybe, it was working before I jumped in and refactored, but it wasn't in the state it's in below...

I like the idea of having examples for our future colleagues/ourselves, but we can keep local copies of cool cypher that never gets used on either in the code base or in production.